src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -60,6 +60,12 @@
 						</template>
 						{{ t('forms', 'Copy to clipboard') }}
 					</NcActionLink>
+					<NcActionButton @click.prevent="openQrModal(getPublicShareLink(share))">
+						<template #icon>
+							<IconQr :size="20" />
+						</template>
+						{{ t('forms', 'Show QR-Code') }}
```

Comment:
```suggestion
						{{ t('forms', 'Show QR code') }}
```
In English it's without the hyphen and a lowercase 'c'